### PR TITLE
Fix duplicated bug and batch upload

### PR DESCRIPTION
**Pull request by gregorjerse**
Otherwise exception is thrown.

### CODE REVIEW

File: resolwe/storage/connectors/googleconnector.py [hunk 1]
```diff
@@ -52,13 +52,16 @@ def __getattr__(self, name):
...
-            with self.client.batch():
-                for delete_url in urls:
-                    blob = self.bucket.blob(
-                        os.fspath(self.base_path / url / delete_url)
-                    )
-                    if blob.exists():
+        # At most 1000 objects can be deleted at the same time.
+        max_chunk_length = 1000
+        for i in range(0, len(urls), max_chunk_length):
+            with suppress(NotFound):
```
Comment (dblenkus) [inline resolwe/storage/connectors/googleconnector.py:58]: Is it ok to put `suppress` here? Will be the rest of 1000 objects when one fails? Would it make more sense to put it only around `delete`?

File: resolwe/flow/executors/__main__.py [hunk 2]
```diff
@@ -98,9 +117,16 @@ async def run_executor():
...
+        try:
+            # Try to obtains exclusive lock over stdout and jsonout files.
+            # When lock can not be obtained this task is probably a duplicate
+            # spawned by celery so lock error and skip processing.
+            log_file, json_file = get_stdout_json_file()
+            await transfer_data()
+            await run_executor(log_file, json_file)
+            await collect_files()
+        except FileExistsError:
+            logger.error("Stdout or jsonout file already exists, aborting.")
```
Comment (dblenkus) [inline resolwe/flow/executors/__main__.py:129]: Do you need to send `ABORT` command to the listener to keep the semaphore in sync?

File: resolwe/storage/connectors/googleconnector.py (hunk 1 above)
Comment (gregorjerse) [reply resolwe/storage/connectors/googleconnector.py:58]: The documentation is not really clear but according to my experience all existing objects in the batch is deleted and the exception for last non-existing element in a batch is raised.

File: resolwe/flow/executors/__main__.py (hunk 2 above)
Comment (gregorjerse) [reply resolwe/flow/executors/__main__.py:129]: I believe not since executors has not even started at this point and no messages have been exchanged.

Comment (gregorjerse) [reply resolwe/flow/executors/__main__.py:129]: I will take another look into that.

Comment (dblenkus) [reply resolwe/flow/executors/__main__.py:129]: It is increased here: https://github.com/genialis/resolwe/blob/fcbf83ac008d5fc039f4e0e69c00fc2e4b439fc0/resolwe/flow/managers/dispatcher.py#L429

Comment (gregorjerse) [reply resolwe/flow/executors/__main__.py:130]: Yes, it is. So to keep the counters in sync the abort has to be called.

**Pull request merged**